Assets/Scripts/ApplicationStateManager.cs

```diff
@@ -12,6 +13,11 @@ public class ApplicationStateManager : MonoBehaviour
     private Unit[] units;
     private float winLossTimer;
     public float eliminationTime;
+    public int handSize = 2;
+    bool pauseFix; //makes it so player doesn't use spell when pressing 'A' in the pause menu to resume game.
```

Comment:
I'm still not entirely comfortable with this implementation. It seems to me we ought to be designing some way to categorize game UI elements from menu ones, and only pausing the game UI, not flipping a bool around like this.

Reply:
I suggested to Brett that he try using a transparent overlay behind the menu that's part of the UI group. That way, maybe we can keep using the EventSystem.current.IsPointerOverGameObject() function to ignore button UI elements (or something similar that is usable with controller software cursors). That's what we were previously doing inside Player.UseCard to ignore the on-screen UI button press actions.